server/database/migrations/20190610103319_create_languages.js

```diff
@@ -1,8 +1,11 @@
-
 exports.up = function(knex, Promise) {
-  
+  return knex.schema.createTable('languages', (table) => {
+    table.increments();
+    table.string('name', 50).notNull();
```

Comment:
better to have two fields for this? e.g. id = 2; english_name = 'spanish'; native_name = 'espanol'; ?